identity/src/commonMain/kotlin/com/android/identity/document/Document.kt

```diff
@@ -209,9 +209,7 @@ class Document private constructor(
         var candidate: Credential? = null
         _certifiedCredentials.filter {
             it.domain == domain && (
-                    now != null
-                            && (now >= it.validFrom)
-                            && (now <= it.validUntil)
+                    now == null || (now >= it.validFrom && now <= it.validUntil)
```

Comment:
No, the check has changed. This changes the meaning of null to, I think, what it was intended to be.

Before, this would fail if "now" was null (so no documents would be considered valid), but the documentation indicated that null was supposed to mean "don't check validity times." Now, if it's null, it doesn't do a time check.

Reply:
Nice catch, I missed that twice I guess... when I wrote this and also during this review!